test/e2e/libpod_suite_test.go

```diff
@@ -254,6 +254,36 @@ func (s *PodmanSession) OutputToStringArray() []string {
 	return strings.Split(output, "\n")
 }
 
+// ErrorGrepString takes session stderr output and behaves like grep. it returns a bool
+// if successful and an array of strings on positive matches
+func (s *PodmanSession) ErrorGrepString(term string) (bool, []string) {
+	var (
+		greps   []string
+		matches bool
+	)
+
+	for _, line := range strings.Split(s.ErrorToString(), "\n") {
+		if strings.Contains(line, term) {
+			matches = true
+			greps = append(greps, line)
+		}
+	}
+	return matches, greps
+}
+
+// ErrorToString formats session stderr to string
+func (s *PodmanSession) ErrorToString() string {
+	fields := strings.Fields(fmt.Sprintf("%s", s.Err.Contents()))
+	return strings.Join(fields, " ")
+}
+
+// ErrorToStringArray returns the stderr output as a []string
+// where each array item is a line split by newline
+func (s *PodmanSession) ErrorToStringArray() []string {
+	output := fmt.Sprintf("%s", s.Err.Contents())
```

Comment:
Why not just use stirngs.Fields here?

Reply:
Because I just ported it from OutputToStringArray. I can fix both on this if you want.

Reply:
Sure, we should try and avoid duplicating the standard library where we can

Reply:
So the test suite expects everything to be split by new line. fields splits by white space and this change breaks one of the tests. Is there a reason it was broken by new line/is it worth making the changes to the whole test suite to use the standard library and save two lines of code?

Reply:
One of the tests ~so far~, I only am running the search test suite and I'm sure it breaks more..

Reply:
@haircommander If it breaks the tests, it's not worth it, then